apps/studio/components/interfaces/Functions/EdgeFunctionsListItem.tsx

```diff
@@ -29,12 +30,15 @@ export const EdgeFunctionsListItem = ({ function: item }: EdgeFunctionsListItemP
       ? `https://${customDomainData.customDomain.hostname}/functions/v1/${item.slug}`
       : `${protocol}://${endpoint}/functions/v1/${item.slug}`
 
+  const handleNavigation = createNavigationHandler(`/project/${ref}/functions/${item.slug}`, router)
+
   return (
     <TableRow
       key={item.id}
-      onClick={() => {
-        router.push(`/project/${ref}/functions/${item.slug}`)
-      }}
+      onClick={handleNavigation}
```

Comment:
This won't be screen-reader friendly because the `TableRow` can't take a different aria role without interfering with table semantics. So screen reader users won't have any way to click the row. In the `BucketTable`, we deal with this by adding a hidden button:

```
      <BucketTableCell>
        <div className="flex justify-end items-center h-full">
          <ChevronRight aria-hidden={true} size={14} className="text-foreground-muted/60" />
        </div>
        <button tabIndex={-1} className="sr-only">
          Go to bucket details
        </button>
      </BucketTableCell>
```